apps/src/music/views/ExemplarPlayerView.tsx

```diff
@@ -30,15 +30,26 @@ const ExemplarPlayerView: React.FunctionComponent<ExemplarPlayerViewProps> = ({
   const dispatch = useAppDispatch();
   const isPlaying = useAppSelector(state => state.music.isPlaying);
   const [exemplarIsPlaying, setExemplarIsPlaying] = useState<boolean>(false);
+  const analyticsReporter = useRef<AnalyticsReporter>(new AnalyticsReporter());
```

Comment:
Instead of instantiating a new reporter, I think you should be able to grab this out of the `AnalyticsContext` - or even just pass it in as a prop from `MusicLabView`